### DISCUSSION TITLE
db.json可以删掉吗?

### DISCUSSION THREAD

**Discussion by cohhi** [hexojs/hexo#5274] [Question / Answer] [answered]
创建文章后,hexo目录的db.json会产生博客站点配置和数据的文件
但我删除db.json后本地查看博客并没有什么问题
请问这个文件可以删除吗

**Comment by li-nuonuo** [comment]
可以删,但不建议,生成静态文件或本地查看时hexo会自动重新创建`db.json`文件

**Comment by uiolee** [comment, marked as answer]
`db.json` can be deleted. 

`hexo clean` is equivalent to removing `db.json` and `public/`

https://hexo.io/docs/commands.html#clean
